subprojects/docs/docs.gradle

```diff
@@ -377,7 +377,7 @@ tasks.withType(AsciidoctorTask).configureEach {
         'samples-dir'       : "$projectDir/src/samples"
```

Comment:
Why don't we just use `src/samples` here instead of using `$projectDir/src/samples` and then relativizing it again?

Reply:
That's what I was about to ask @eriwen . Where is `samples-dir` used? Do we need absolute path here?

Reply:
I'm not sure that is used at all. I cannot find any usages in Gradle or Exemplar.

Reply:
It is used here: https://github.com/gradle/dotorg-docs/blob/5898ed8f1aadf4793cac6fe934548d73d37e62b7/docs-asciidoctor-extensions/src/main/java/org/gradle/docs/asciidoctor/SampleIncludeProcessor.java#L76

Reply:
Ah... thanks... it's private repo, no wonder I can't search it...

Reply:
D'oh, that was an obvious miss on my part.